pcap/pcap.h

```diff
@@ -343,6 +343,7 @@ PCAP_API const char *pcap_tstamp_type_val_to_description(int);
 
 #ifdef __linux__
 PCAP_API int	pcap_set_protocol(pcap_t *, int);
+PCAP_API int    pcap_set_fanout(pcap_t *, int, int);
```

Comment:
I think there's one extra space between `int` and the function name, so that the names don't line up.